system_metrics_collector/test/system_metrics_collector/test_linux_process_cpu_measurement_node.cpp

```diff
@@ -77,147 +88,12 @@ class MockLinuxProcessCpuMeasurementNode : public system_metrics_collector::
 private:
   system_metrics_collector::ProcPidCpuData MakeSingleMeasurement() override
   {
-    EXPECT_GT(test_constants::kProcPidSamples.size(), measurement_index_);
-    return test_constants::kProcPidSamples[measurement_index_++];
+    EXPECT_GT(kProcPidSamples.size(), measurement_index_);
```

Comment:
```suggestion
    ASSERT_GT(kProcPidSamples.size(), measurement_index_);
```
Since the following line will index out of bounds if the `EXPECT_EQ` fails.

Reply:
Could use this if the return type was void.